src/storekit.cs

```diff
@@ -6,7 +6,7 @@
 //   Miguel de Icaza
 //
 // Copyright 2009, Novell, Inc.
-// Copyright 2012 Xamarin Inc.
+// Copyright 2020 Xamarin Inc.
```

Comment:
don't remove, replace - just append
in this case it should be a new line for 2020 / Microsoft